sockets.go

```diff
@@ -34,27 +34,18 @@ import (
 	"time"
 )
 
+var WriteWait = time.Second * 10
```

Comment:
This can be a little more DRY by using the following notation:

`           
  
    var (
	    // Time allowed to write a message to the peer.
 	    writeWait = 10 * time.Second

 	    // Time allowed to read the next pong message from the peer.
 	    pongWait = 60 * time.Second

 	    // Send pings to peer with this period. Must be less than pongWait.
 	    pingPeriod = (pongWait * 9) / 10

 	    // Maximum message size allowed from peer.
        maxMessageSize = 2560
    )
`